src/app/modules/myinfo/myinfo.controller.ts

```diff
@@ -151,22 +151,19 @@ const validateMyInfoLogin = celebrate({
       .unknown(true),
     Joi.object()
       .keys({
-        // TODO (#1211): determine whether production returns error_description or error-description
-        error_description: Joi.string(),
         'error-description': Joi.string(),
         error: Joi.string().required(),
         state: Joi.string().required(),
       })
-      .xor('error-description', 'error_description'),
+      // Allow other params in case MyInfo adds them in future
+      .unknown(true),
```

Comment:
I don't think this is necessary, since the keys are bound by API contract and can't be changed.

Reply:
I think this would also defeat the point of Joi validation for security

Reply:
> // MyInfo sends several other params which are not necessary for Form

Never mind, just saw this.